### DISCUSSION TITLE
Copilot user change not possible on ubuntu

### DISCUSSION THREAD

**Discussion by umitnrodemund** [community/community#77036] [Copilot Conversations]
### Select Topic Area

Question

### Body

I accidentally logged in into the wrong github account. I cannot change that now on ubuntu/visual studio code. The extension seems to store the login information at a place that is not purged on uninstall. How to properly reset the extension when reinstall has no effect?

What I tried:
- Uninstall, restart Code, reinstall
- Logout of all github accounts on all browsers
- Being in the right account while uninstall/retry

Message:
[INFO] [auth] [2023-11-26T15:40:59.694Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-11-26T15:40:59.694Z] Extension activation failed: "Thank you for using GitHub Copilot. Your subscription has ended. You are currently logged in as n*** (wrong account)

**Comment by JonaEscobar** [comment]
Hi,
Try with Accounts (on the bottom left corner), then sign out, then you can log in with another user, see attached image
<img width="288" alt="Capture" src="https://github.com/community/community/assets/134455051/f8af2858-204d-4740-8185-91f146f432de">

regards,
